src/openrct2/rct2/S6Importer.cpp

```diff
@@ -701,6 +703,36 @@ class S6Importer final : public IParkImporter
     {
         game_init_all(_s6.map_size);
     }
+
+    /**
+     * Clears incorrectly set guest entry points.
+     */
+    void FixEntryPoints()
+    {
+        // While we probably could fix both saves and scenarios, I want to avoid touching saves where the user
+        // might already have fixed this, or just happens to share a name.
```

Comment:
If you write in 1st person, you should probably include a signature.

Reply:
I can write it in third person, if you can get behind my reasoning here.

Reply:
ok